rest-service/manager_rest/cluster_status_manager.py

```diff
@@ -123,20 +123,32 @@ def get_cluster_status(detailed=False):
     return cluster_status
 
 
-def _add_monitoring_data(cluster_nodes):
-    # Get all the data in one fell swoop (thanks to the underlying Prometheus
-    # federation).
+def _add_monitoring_data(cluster_nodes: dict):
```

Comment:
If you're adding the parameter type, it might be good to add the return type (which I think in this case is -> None?)